### ISSUE TITLE
tsconfig.json:18:5 - error TS5053: Option 'sourceMap' cannot be specified with option 'inlineSourceMap'.

### ISSUE THREAD

**Issue by Drew-Daniels** [infinitered/gluegun#786]
**Describe the bug**
After following the instructions on the [Quick Start](https://github.com/infinitered/gluegun#quick-start), I am able to successfully link and run a newly created CLI app, but am running into an issue running `yarn build`, without having modified any of the originally generated code from running the `gluegun new` script.

**To Reproduce**
1. Create a CLI with `npx gluegun new my-app`
2. Select TypeScript as language
3. Run `yarn build`
4. See TypeScript build error

![image](https://github.com/infinitered/gluegun/assets/62859552/3ba13328-70bb-4d66-95f3-fd8302453195)

**Expected behavior**
I would expect that upon trying to build the app after not modifying any of the source code, the build would work.

**Screenshots**

**Doctor (please complete the following information):**
 - OS: [macOS 14.0 (23A344)]
 - Gluegun Version [5.1.6]

**Additional context**
Removing `"sourceMap": true` from my `tsconfig.json` fixed the issue - [link](https://stackoverflow.com/a/42042292/13175926)

I have a globally installed `typescript v5.2.2`.

**Comment by jamonholmgren** [comment]
Yeah we need to remove one of them. Any recommendations which? I can look later but currently on mobile.

**Comment by Drew-Daniels** [comment]
Thanks for confirming! My vote would be for removing `sourceMap` since it would result in less files being output as part of the build (simplicity), but I wouldn't say I have a huge preference either way.
